src/taipy/core/config/scenario_config.py

```diff
@@ -169,12 +169,12 @@ def _from_dict(
             task_config_ids = as_dict.pop(cls._TASKS_KEY, list())
             tasks = cls.__get_task_configs(task_config_ids, config)
         else:
-            pipeline_config_ids = as_dict.pop(cls._PIPELINES_KEY, list())
+            sequence_config_ids = as_dict.pop(cls._SEQUENCES_KEY, list())
             cls.__logger.info(
-                f"The tasks from these PipelineConfig {pipeline_config_ids}"
+                f"The tasks from these SequenceConfig {sequence_config_ids}"
                 f"will be migrated to be directly under ScenarioConfig {id}"
             )
-            tasks = cls.__get_task_configs_from_pipeline_configs(pipeline_config_ids, config)
+            tasks = cls.__get_task_configs_from_sequence_configs(sequence_config_ids, config)
```

Comment:
I have a question. If we rename the PipelineConfig to SequenceConfig, then user's old config should not work anymore. So is this `else` block necessary anymore?

Reply:
that's true. the code for migrating from old configs to new configs are no longer valid. @jrobinAV I think we will need to discuss about this more

Reply:
either we need a new logic for migrating old PipelineConfig to new SequenceConfig, or we remove it in core and move this logic to migration script. Either way, we will still need new logic as Scenario is still using the PipelineConfig to pull task configs when none is provided. And regardless, of the solution, I think we can implement it on this PR instead https://github.com/Avaiga/taipy-core/pull/739. To keep this PR strictly for renaming only with no logic changed. Let me know what you guys think @trgiangdo @jrobinAV